### PR TITLE
Springer update 2

### PR DESCRIPTION
**Pull request by rmzelle**
In this pull request, I overwrote the BMC styles (generated with `data/bmc`) with styles generated from the new Springer metadata.

I deleted any styles not generated from the Springer metadata. These are:

Discontinued:
BMC Medical Physics, http://www.biomedcentral.com/bmcmedphys/
Herpesviridae, http://www.herpesviridae.org/
Longevity & Healthspan, http://www.longevityandhealthspan.com/
Microbial Informatics and Experimentation, http://www.microbialinformaticsj.com/

Transferred:
Genome Integrity, was http://www.genomeintegrity.com/, now http://www.genome-integrity.org/. Currently published by Medknow (Wolters Kluwer Health).
International Archives of Medicine, http://www.intarchmed.com/. Relaunced? ("IAM 2.0") Now published by iMed.pub.
Journal of Brachial Plexus and Peripheral Nerve Injury, now http://www.thieme.com/jbppni. Currently published by Thieme Publishers.
Journal of Molecular Signaling, http://www.jmolecularsignaling.com/. Currently published by Ubiquity Press.

Other:
BMC Proceedings, http://www.biomedcentral.com/bmcproc/ (absent because not a regular journal?)

### CODE REVIEW

**Comment by rmzelle** [comment]
It looks like most, if not all styles have transitioned from biomed-central as the parent to one of the Springer styles. We should check a few. We also need to update renamed-styles.json to redirect the deleted styles.

File: dependent/acta-veterinaria-scandinavica.csl [hunk 1]
```diff
@@ -1,18 +1,16 @@
...
-  <!-- Generated with https://github.com/citation-style-language/utilities/tree/master/generate_dependent_styles/data/bmc -->
+  <!-- Generated with https://github.com/citation-style-language/utilities/tree/master/generate_dependent_styles/data/springer -->
   <info>
     <title>Acta Veterinaria Scandinavica</title>
-    <title-short>Acta Vet Scand</title-short>
     <id>http://www.zotero.org/styles/acta-veterinaria-scandinavica</id>
     <link href="http://www.zotero.org/styles/acta-veterinaria-scandinavica" rel="self"/>
-    <link href="http://www.zotero.org/styles/biomed-central" rel="independent-parent"/>
-    <link href="http://www.actavetscand.com" rel="documentation"/>
+    <link href="http://www.zotero.org/styles/springer-basic-brackets" rel="independent-parent"/>
```
Comment (rmzelle) [inline dependent/acta-veterinaria-scandinavica.csl]: Seems wrong. http://www.actavetscand.com/authors/instructions/research#formatting-references links to "SpringerVancouverNumber.ens", and examples look more like springer-vancouver-brackets.csl. (the original "biomed-central.csl" is wrong too)

File: dependent/addiction-science-and-clinical-practice.csl [hunk 2]
```diff
@@ -1,18 +1,16 @@
...
-  <!-- Generated with https://github.com/citation-style-language/utilities/tree/master/generate_dependent_styles/data/bmc -->
+  <!-- Generated with https://github.com/citation-style-language/utilities/tree/master/generate_dependent_styles/data/springer -->
   <info>
     <title>Addiction Science &amp; Clinical Practice</title>
-    <title-short>Addict Sci Clin Pract</title-short>
     <id>http://www.zotero.org/styles/addiction-science-and-clinical-practice</id>
     <link href="http://www.zotero.org/styles/addiction-science-and-clinical-practice" rel="self"/>
-    <link href="http://www.zotero.org/styles/biomed-central" rel="independent-parent"/>
-    <link href="http://www.ascpjournal.org" rel="documentation"/>
+    <link href="http://www.zotero.org/styles/springer-basic-brackets" rel="independent-parent"/>
```
Comment (rmzelle) [inline dependent/addiction-science-and-clinical-practice.csl]: http://www.ascpjournal.org/authors/instructions/research#formatting-references. Also looks like springer-vancouver-brackets.csl

File: dependent/agriculture-and-food-security.csl [hunk 3]
```diff
@@ -1,18 +1,16 @@
...
-  <!-- Generated with https://github.com/citation-style-language/utilities/tree/master/generate_dependent_styles/data/bmc -->
+  <!-- Generated with https://github.com/citation-style-language/utilities/tree/master/generate_dependent_styles/data/springer -->
   <info>
     <title>Agriculture &amp; Food Security</title>
-    <title-short>Agric Food Secur</title-short>
     <id>http://www.zotero.org/styles/agriculture-and-food-security</id>
     <link href="http://www.zotero.org/styles/agriculture-and-food-security" rel="self"/>
-    <link href="http://www.zotero.org/styles/biomed-central" rel="independent-parent"/>
-    <link href="http://www.agricultureandfoodsecurity.com" rel="documentation"/>
+    <link href="http://www.zotero.org/styles/springer-basic-brackets" rel="independent-parent"/>
```
Comment (rmzelle) [inline dependent/agriculture-and-food-security.csl]: http://www.agricultureandfoodsecurity.com/authors/instructions/research#formatting-references. Also looks like springer-vancouver-brackets.csl

Comment (zuphilip) [inline dependent/agriculture-and-food-security.csl]: Yeah, it looks like most of the styles should be connected to the `springer-vancouver-brackets.csl` and not `springer-basic-number.csl`. How can we deal with that?

Moreover, the shorttitle and field(s) are not present anymore. Shouldn't we try to add them again, maybe using a similar approach in Excel as before?

**Comment by zuphilip** [comment]
I looked at some more examplary journals and recently published articles. It looks like mostly the year is appearing at the end and not directly after the authors, i.e. it more looked as Springer-Vancouver than Springer-Basic style. This fits with the instructions for authors normally. However, in the list derived from the Springer database there seems differences, e.g.:
```
BMC Medical Research Methodology	1471-2288		Vancouver	Numbered
BMC Family Practice	1471-2296		Basic	Numbered
BMC Gastroenterology	1471-230X		Basic	Numbered
BMC Geriatrics	1471-2318		Vancouver	Numbered
```
Are these all errors in the Springer database??

**Comment by rmzelle** [comment]
I'm not sure why this is such a mess. I emailed BMC about the double entries in their journal spreadsheet (see http://www.biomedcentral.com/journals/biomedcentraljournallist.txt at http://www.biomedcentral.com/about/catalog) in January (at info@biomedcentral.com), and reached out via Twitter (https://twitter.com/rintzezelle/status/593953257355124737, https://twitter.com/rintzezelle/status/593953473529565185, https://twitter.com/rintzezelle/status/593953745618231297), but never heard back.

Do we have any contacts at Springer that could look into this?

**Comment by zuphilip** [comment]
I can try to reach out at Springer over my contact. I will try to do it on Monday. BTW we might have to call it now "Springer Nature" after their big merge.

**Comment by zuphilip** [comment]
<i>Update:</i> I already received a first answer: it will take some time, because it has to go through several hands within Springer/Biomed Central.

File: dependent/aids-research-and-therapy.csl [hunk 4]
```diff
@@ -1,18 +1,16 @@
...
-  <!-- Generated with https://github.com/citation-style-language/utilities/tree/master/generate_dependent_styles/data/bmc -->
+  <!-- Generated with https://github.com/citation-style-language/utilities/tree/master/generate_dependent_styles/data/springer -->
   <info>
     <title>AIDS Research and Therapy</title>
-    <title-short>AIDS Res Ther</title-short>
     <id>http://www.zotero.org/styles/aids-research-and-therapy</id>
     <link href="http://www.zotero.org/styles/aids-research-and-therapy" rel="self"/>
-    <link href="http://www.zotero.org/styles/biomed-central" rel="independent-parent"/>
-    <link href="http://www.aidsrestherapy.com" rel="documentation"/>
+    <link href="http://www.zotero.org/styles/springer-basic-brackets" rel="independent-parent"/>
```
Comment (zuphilip) [inline dependent/aids-research-and-therapy.csl]: Official answer for this journal: Vancouver is correct. I will aks again if this is somehow in general true.

Comment (rmzelle) [inline dependent/aids-research-and-therapy.csl]: I guess Vancouver means "springer-vancouver-brackets.csl" here?

Comment (zuphilip) [inline dependent/aids-research-and-therapy.csl]: Yeah, it should be numbered (i.e. -brackets) I guess...

**Comment by zuphilip** [comment]
A representative from Biomed Central wrote: 
<blockquote>All journals that were previously BioMed Central (BMC) Reference Style or Springer Open Reference Style will have now changed to Vancouver. Any of our journals that were not previously using one of these reference styles will have stayed the same. No journals have changed from BMC to Springer style.</blockquote>

I conclude that:
 a) We can generalize the answer from one journal previously using BMC style to all journals previously using BMC style.
 b) Replacing with Springer-Basic-Brackets is wrong
 c) We have to replace it with <i>some</i> Vancouver style.

In the instructions for authors for the one journal: http://www.aidsrestherapy.com/authors/instructions/research#formatting-references it is linked to SpringerVancouverNumber.ens. Thus, it looks like Springer-Vancouver-Brackets is the style we should replace with. This looks like the solution; what do you think @rmzelle ? Any open questions left?

**Comment by rmzelle** [comment]
Seems pretty clear. Too bad the metadata is such a mess, though. But we probably want to get rid of the "bmc" folder and rely solely on the "springer" folder for generating these styles.

**Comment by adam3smith** [comment]
superseded by https://github.com/citation-style-language/styles/pull/1885

**Pull request closed without merging**